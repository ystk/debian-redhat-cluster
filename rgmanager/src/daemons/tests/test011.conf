<?xml version="1.0"?>
<!-- Add NFS clients to the mix.
     Between test 10 and this one, there should be NEEDSTARTS in the
     new resource tree for Dummy Export.

     Because everything below an element is started if there's a NEEDSTART
     flag (and everything below an element is stopped if there's a
     NEEDSTOP, too), Admin group and User group will not need NEEDSTART
     flags in the resource trees, but their output in the resource list
     should show that they do have NEEDSTART flags.
-->
<cluster>
<rm>
	<resources>
    		<service name="test1"/>
		<script name="initscript" file="/etc/init.d/sshd"/>
		<ip address="192.168.1.3" monitor_link="yes"/>
    		<fs fstype="ext3" name="mount1" mountpoint="/mnt/cluster" device="/dev/sdb8"/>
		<nfsexport name="Dummy Export"/>
    		<nfsclient name="User group" target="@users" options="ro"/>
    		<nfsclient name="Admin group" target="@admin" options="rw"/>
    		<nfsclient name="yellow" target="yellow" options="rw,no_root_squash"/>
    		<nfsclient name="magenta" target="magenta" options="rw,no_root_squash"/>
    		<nfsclient name="red" target="red" options="rw,no_root_squash"/>
		
	</resources>
	<service ref="test1">
		<ip ref="192.168.1.3"/>
		<script ref="initscript"/>
		<fs ref="mount1">
			<nfsexport ref="Dummy Export">
				<nfsclient ref="Admin group"/>
				<nfsclient ref="User group"/>
			</nfsexport>
		</fs>
	</service>
</rm>
</cluster>
