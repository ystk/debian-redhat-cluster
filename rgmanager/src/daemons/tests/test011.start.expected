Starting test1...
[start] service:test1
[start] fs:mount1
[start] nfsexport:Dummy Export
[start] nfsclient:Admin group
[start] nfsclient:User group
[start] ip:192.168.1.3
[start] script:initscript
Start of test1 complete
